stem/IO/kratos_io.py

```diff
@@ -962,11 +1016,13 @@ def write_project_parameters_json(self, model: Model, mesh_file_name: str,
         outputs_dict = self.__create_output_process_dictionary(outputs=model.output_settings)
         # get the boundary condition dictionary
         loads_and_bc_dict = self.__create_loads_and_boundary_conditions_dictionary(model=model)
+        # get the auxiliary processes dictionary
```

Comment:
why are the mass spring system called auxiliar?

Reply:
the mass and spring systems are still elements (body model parts), however a different process was needed to assign the parameters to the elements. This process is implemented in StemKratos